app/src/main/java/com/cortado/minionify/WelcomeActivity.java

```diff
@@ -0,0 +1,32 @@
+package com.cortado.minionify;
+
+import android.os.Bundle;
+import android.view.Window;
+import android.view.WindowManager;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class WelcomeActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        Window window = getWindow();
+        setActionBarTransparent(window);
```

Comment:
you can remove the action bar completely. https://stackoverflow.com/questions/8456835/how-to-disable-action-bar-permanently

Reply:
Well, I think I had a naming mistake here. This function is to make the **status bar** translucent. The action bar is removed, because the activity's theme is NoActionBar (only the activity's, not the application's).

Reply:
i see, naming is fixed so this is fine